data/genesis/demo-marketplace.toml

```diff
@@ -0,0 +1,43 @@
+base_version = "0.2"
+upgrade_version = "0.3"
+
+[stake_table]
+capacity = 10
+
+[chain_config]
+chain_id = 999999999
+base_fee = '0 wei'
+max_block_size = '1mb'
+fee_recipient = '0x0000000000000000000000000000000000000000'
+fee_contract = '0xa15bb66138824a1c7167f5e85b957d04dd34e468'
+
+[header]
+timestamp = "1970-01-01T00:00:00Z"
+
+[[upgrade]]
```

Comment:
We can remove the fee upgrade from this toml file

Reply:
Is `timestamp` part of the upgrade?

Reply:
No, I meant to comment on the upgrade L17